pkg/elemental/models/elemental.cattle.io.machineinventory.js

```diff
@@ -97,4 +105,14 @@ export default class MachineInventory extends ElementalResource {
       return this.$rootGetters['i18n/t']('resourceTable.groupLabel.notInACluster');
     }
   }
+
+  get cannotBeDeletedByUi() {
+    let cannotBeDeletedByUi = false;
+
+    if (this.status?.conditions && this.status?.conditions.find(c => c.type === 'AdoptionReady' && c.status === 'True')) {
```

Comment:
Will the first condition work if `conditions` is an array?

Reply:
Afaik `conditions` is always an array and it worked just fine while coding this